scripts/build-ios-simulator.sh

```diff
@@ -2,6 +2,8 @@
 
 APP_NAME="Woof"
 xcodebuild -project Woof/${APP_NAME}.xcodeproj \
```

Comment:
nit:
You can use a constant here instead the "Woof" substring.

Reply:
Fixed with https://github.com/ios-course/ironfoudation-team-project/commit/e65e3269a0ac9e61c73db68409846e8f4db93658